src/generators/typescript/renderers/EnumRenderer.ts

```diff
@@ -36,13 +36,59 @@ ${this.indent(this.renderBlock(content, 2))}
   runItemPreset(item: any): Promise<string> {
     return this.runPreset('item', { item });
   }
+
+  normalizeKey(value: any): any {
+    let key;
+    switch (typeof value) {
+    case 'bigint':
+    case 'number': {
+      key = `number ${value}`;
+      break;
+    }
+    case 'object': {
+      key = JSON.stringify(value);
+      break;
+    }
+    default: {
+      key = String(value);
+    }
+    }
+    return FormatHelpers.toConstantCase(key);
+  }
+
+  normalizeValue(value: any): any {
+    let normalizedValue;
+    switch (typeof value) {
+    case 'string':
+      normalizedValue = `"${value}"`;
+      break;
+    case 'boolean':
+      normalizedValue = `"${value}"`;
```

Comment:
```suggestion
      normalizedValue = `${value}`;
```

Reply:
We are not allowed to mix and match booleans in enums... So they have to be rendered as a string rather than the value, therefore the wrap of `"`.

Reply:
I thought that in TS it's possible to use the boolean in enums, sorry 😅

Reply:
Thought the same 😄